onnx_tf/handlers/backend/instance_normalization.py

```diff
@@ -31,7 +31,7 @@ def _common(cls, node, **kwargs):
     beta = tensor_dict[node.inputs[2]]
 
     inputs = tensor_dict[node.inputs[0]]
-    inputs_shape = inputs.shape
+    inputs_shape = tf.shape(inputs)
```

Comment:
We have a common util function call "tf_shape" in onnx-tensorflow/onnx_tf/common/tf_helper.py that would check the shape of the input is fully defined or not to decide to use numpy or tensorflow op. May you please use our "tf_shape" instead of "tf.shape".

Reply:
Thanks! I've made the suggested changes. Please let me know if anything else needs to change.